tests/src/test/java/org/wso2/micro/gateway/tests/context/ServerInstance.java

```diff
@@ -405,7 +405,7 @@ private void startServer(String[] args) throws MicroGWTestException {
                 process = Runtime.getRuntime().exec(cmdArgs, null, commandDir);
 
             } else {
-                cmdArray = new String[]{"bash", "bin/" + scriptName, "run"};
+                cmdArray = new String[]{"bash", "bin/" + scriptName, "run" ,"-e", "b7a.log.level=TRACE"};
```

Comment:
Does this mean all test cases running with trace logs? That will be too much logs

Reply:
Yes, Kept it for debugging purposes